src/metabolic_ninja/worker/data.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2018, Novo Nordisk Foundation Center for Biosustainability,
+# Technical University of Denmark.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+from contextlib import contextmanager
+
+import cobra.io
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from ..models import DesignJob
+from ..universal import UNIVERSAL_SOURCES
+
+
+logger = logging.getLogger(__name__)
+
+
+@contextmanager
+def db_session():
+    """Connect to the database and yield an SA session."""
+    engine = create_engine(
+        "postgresql://{POSTGRES_USERNAME}:{POSTGRES_PASS}@{POSTGRES_HOST}:"
+        "{POSTGRES_PORT}/{POSTGRES_DB_NAME}".format(**os.environ)
+    )
+    Session = sessionmaker(bind=engine)
+    session = Session()
+    yield session
```

Comment:
Now that I look at it again, this should probably be in a `try: ... finally: ...` block.

Reply:
Sounds reasonable to me. Can you create a PR or issue for that, please?

Reply:
https://github.com/DD-DeCaF/scrum/issues/1029